pkgs/development/ocaml-modules/ppx_yojson_conv_lib/default.nix

```diff
@@ -2,17 +2,15 @@
 
 buildDunePackage rec {
   pname = "ppx_yojson_conv_lib";
-  version = "0.15.0";
+  version = "0.16.0";
 
-  useDune2 = true;
-
-  minimumOCamlVersion = "4.02.3";
+  minimalOCamlVersion = "4.14";
```

Comment:
This constraint seems unnecessarily strict.

Reply:
I agree. Reverted.